UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -213,5 +226,86 @@ public async Task WhenGetBatchInfoBasedOnProducts_ThenReturnsSearchBatchResponse
         }
 
         #endregion GetBatchInfoBasedOnProducts
+
+        #region SearchReadMeFilePath
+        [Test]
+        public async Task WhenSearchReadMeFilePath_ThenSearchReadMeFilePathReturnsEmptyStringResponse()
+        {
+            A.CallTo(() => fakeAuthTokenProvider.GetManagedIdentityAuthAsync(A<string>.Ignored)).Returns(GetFakeToken());
+            A.CallTo(() => fakeFileShareServiceClient.CallFileShareServiceApi(A<HttpMethod>.Ignored, A<string>.Ignored, A<string>.Ignored, A<string>.Ignored))
+                 .Returns(new HttpResponseMessage() { StatusCode = HttpStatusCode.BadRequest, RequestMessage = new HttpRequestMessage() { RequestUri = new Uri("http://test.com") }, Content = new StreamContent(new MemoryStream(Encoding.UTF8.GetBytes("Bad request"))) });
+
+            var response = await fileShareService.SearchReadMeFilePath(string.Empty);
+            Assert.AreEqual(string.Empty, response);
+        
+        }
+        [Test]
+        public async Task WhenSearchReadMeFilePath_ThenSearchReadMeFilePathReturnsReadMeFilePathResponse()
```

Comment:
Kindly change WhenSearchReadMeFilePath_ThenSearchReadMeFilePathReturnsReadMeFilePathResponse() to WhenValidRequest_ThenSearchReadMeFileReturnsValidFilePath()

Reply:
Done